src/writer/unfollow.ts

```diff
@@ -1,10 +1,11 @@
 import db from '../helpers/mysql';
+import { defaultNetwork } from './follow';
 
 export async function verify(): Promise<any> {
   return true;
 }
 
 export async function action(message): Promise<void> {
-  const query = 'DELETE FROM follows WHERE follower = ? AND space = ? LIMIT 1';
-  await db.queryAsync(query, [message.from, message.space]);
+  const query = 'DELETE FROM follows WHERE follower = ? AND space = ? AND network = ? LIMIT 1';
+  await db.queryAsync(query, [message.from, message.space, defaultNetwork]);
```

Comment:
why only `defaultNetwork`? users won't be able to unfollow on other networks?

Reply:
i think it should be `message.network || defaultNetwork,` similar to follow action

Reply:
Correct